anya/roman_numeral.rb

```diff
@@ -0,0 +1,36 @@
+class RomanNumeral
```

Comment:
It would make it more extensible, however, we were trying to keep the code simple and avoid premature optimization.